Mathlib/Data/Matrix/Notation.lean

```diff
@@ -181,25 +181,27 @@ end DotProduct
 
 section ColRow
 
+variable {ι : Type*} [Unique ι]
 @[simp]
-theorem col_empty (v : Fin 0 → α) : col v = vecEmpty :=
+theorem col_empty (v : Fin 0 → α) : col (ι := ι) v = vecEmpty :=
```

Comment:
These statements aren't really well-typed, see #11593

Reply:
If I add your modifications from that PR here, I still need the exact same modifications as they are now. I believe these two issues are not directly connected.

Reply:
Yes, no action needed by you; your change just caused me to spot a latent issue in this file, and I thought I'd leave a comment in case another reviewer had the same thought.